src/cmd-cloud-prune

```diff
@@ -141,13 +143,13 @@ def main():
                         # S3().delete_object(args.bucket, f"{args.prefix}{str(current_build.id)}")
                     case "images":
                         raise NotImplementedError
-            if not args.dry_run:
-                build.setdefault("policy-cleanup", []).append("cloud-uploads")
-                builds_json_data["builds"][index] = build
+            build.setdefault("policy-cleanup", []).append("cloud-uploads")
+            builds[len(builds) - 1 - index] = build
 
-    if not args.dry_run:
-        # Save the updated builds.json to local builds/builds.json
-        save_builds_json(builds_json_data)
+    # Ensure the builds are ordered from newest to oldest.
+    builds.sort(key=lambda x: x['id'], reverse=True)
```

Comment:
We shouldn't be doing any sorting at all here. Why do we need this? How is the array getting unsorted in the first place?